lymph/node.py

```diff
@@ -49,12 +51,12 @@ def state(self, newstate: int):
         """Set the state of the node and make sure the state of a tumor node
         cannot be changed."""
         if self.typ == "lnl":
-            if int(newstate) not in [0,1]:
-                raise ValueError("State of node must be either 0 or 1")
+            if int(newstate) not in list(range(self.allowed_states)):
+                raise ValueError("State of node must be either 0, 1,... allowed_states - 1")
             self._state = int(newstate)
 
         elif self.typ == "tumor":

```

Comment:
Maybe we can create two subclasses of `Node`: One `LNLNode` and one `TumorNode` instead of this "hacky" `typ` parameter I used so far...

Reply:
This would make sense, I have been thinking about how to change the problem of assigning growth to the tumor node, where no growth should be present.

Reply:
Or use the "circular" edges 😉